tasks/src/reinforcement-learning/data.ts

```diff
@@ -0,0 +1,35 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "A subset of widely used Datasets for Data Driven Deep Reinforcement Learning (D4RL)",
+			id:          "edbeeching/decision_transformer_gym_replay",
+		}
+	],
+    demo:     {
+		inputs:  [],
+		outputs: [],
+	},
+    metrics: [{
+        description: "Mean reward per episode obtained after running policy for a certain number of evaluation episodes ",

```

Comment:
I was wondering if you'd like to leave some terms such as episode and policy in `about.md` file? There's a section called About the Task that you could leave a Glossary there! it would help a ton!